### PR TITLE
OCPBUGS-35851#adding missing port details

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version:
4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-35851

Link to docs preview:
https://79494--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/ipi/installing-azure-government-region.html#installation-about-custom-azure-vnet-nsg-requirements_installing-azure-government-region

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@jinyunma, PTAL at this draft update.

**Comment by jinyunma** [comment]
@brendan-daly-red-hat thanks for addressing this.
LGTM

File: modules/installation-about-custom-azure-vnet.adoc [hunk 1]
```diff
@@ -130,6 +130,73 @@ include::snippets/mcs-endpoint-limitation.adoc[]
...
+|`500`
+|IPsec IKE packets
+
+|`4500`
+|IPsec NAT-T packets
+
+|`123`
+|Network Time Protocol (NTP) on UDP port `123`
+
+If an external NTP time server is configured, you must open UDP port `123`.
```
Comment (xenolinux) [inline modules/installation-about-custom-azure-vnet.adoc:175]:
```suggestion
If you configure an external NTP time server, you must open UDP port `123`.
```

**Review by xenolinux — COMMENTED** [review]
One minor comment; otherwise LGTM

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by jab-rh** [comment]
@brendan-daly-red-hat, looks like this doesn't apply cleanly to 4.16 and prior. Can you manually cherry-pick this? It might cherry-pick back from 4.16 if fixed. Thanks!

**Comment by jab-rh** [comment]
@brendan-daly-red-hat thanks!

**Pull request merged**